docs/src/modules/components/DemoToolbar.js

```diff
@@ -741,6 +707,8 @@ export default function DemoToolbar(props) {
 DemoToolbar.propTypes = {
   codeOpen: PropTypes.bool.isRequired,
   codeVariant: PropTypes.string.isRequired,
+  copyButtonOnClick: PropTypes.object.isRequired,
```

Comment:
Wrong type fixed in https://github.com/mui/material-ui/pull/41994#discussion_r1631016976